addons/auth_passkey_portal/security/security.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.rule" id="rule_auth_passkey_key_portal">
+        <field name="name">Passkeys: Portal users can only access own Passkeys</field>
+        <field name="model_id" ref="auth_passkey.model_auth_passkey_key"/>
+        <field name="groups" eval="[(4, ref('base.group_portal'))]"/>
```

Comment:
I think we should just add `group_portal` to the existing security rule `rule_auth_passkey_key_user`

For instance this is what `auth_totp` does. It does not add a new rule in `auth_totp_portal` but rather in the `auth_totp` module itself `group_portal` is added to the security rule

https://github.com/odoo/odoo/blob/0abdcd9ef6ad3fc932dc0eb46d8aa973b00c34c2/addons/auth_totp/security/security.xml#L29